alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/accessor/SettingsUtility.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2019 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.descriptor.accessor;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+import com.synopsys.integration.alert.common.exception.AlertException;
+import com.synopsys.integration.alert.common.persistence.model.ConfigurationFieldModel;
+import com.synopsys.integration.alert.common.persistence.model.ConfigurationModel;
+import com.synopsys.integration.alert.common.persistence.model.DefinedFieldModel;
+
+public interface SettingsUtility {
+
+    String getSettingsName();
```

Comment:
I was questioning adding this item to this interface. I feel this utility should perform the actions necessary instead of letting people figure out the name and doing it themselves. Should we keep this? There were a couple classes that would require a bit of a rework if they didn't know the settings descriptor name.

Reply:
Why not have the universal key?